conformance/test/client.py

```diff
@@ -518,9 +520,6 @@ async def send_unary_request(
             test_response.response.error.message = e.message
             test_response.response.error.details.extend(e.details)
         except (asyncio.CancelledError, Exception) as e:
-            import sys
```

Comment:
Thanks for fixing lint with updated ruff, forgot to update it. I think actually most of these weren't intentional, so I restored the rule